### PR TITLE
TW 107 and 108

### PR DESCRIPTION
**Pull request by DJSanti**
## What does this PR do?

This PR covers TW-107 and TW-108 concerning the Web UI page on docs.temporal.io.
The page has been reviewed for prose, and a note at the beginning was removed due to the UI long being updated.

### CODE REVIEW

**Comment by DJSanti** [comment]
yarn gen issues...

File: docs-src/concepts/what-is-the-temporal-web-ui.md [hunk 1]
```diff
@@ -31,31 +23,35 @@ The Web UI is packed with several features.
 
 ### Namespace selection
 
-Users can switch between Namespaces by clicking the Namespace Switcher icon in the left navigation.
-Clicking the Namespace Switcher reveals a side panel where users can search for a Namespace or select a Namespace from the list.
-After a Namespace is selected, the user is redirected to the Recent Workflows page for that Namespace.
-In Temporal Cloud, users can access only Namespaces that they have been granted access to.
+The Namespace Switcher (located in the left navigational menu) allows users to search for or select a Namespace.
+After a Namespace is selected, the user is redirected to that Namespace's Recent Workflows.
+In Temporal Cloud, users can only access Namespaces that they have been granted access to.
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-web-ui.md:28]:
```suggestion
You can search for or select a Namespace by using the Namespace Switcher, located in the left navigational menu.
After you select a Namespace, the Web UI shows the Recent Workflows page for that Namespace.
In Temporal Cloud, users can access only the Namespaces that they have been granted access to.
```

File: docs-src/concepts/what-is-the-temporal-web-ui.md [hunk 2]
```diff
@@ -31,31 +23,35 @@ The Web UI is packed with several features.
...
 
-The Recent Workflows page lists the most recent 1,000 Workflow Executions in the past 24 hours, by default.
+Recent Workflows lists all Workflow Executions run in the past 24 hours.
+The default number shown is 1,000 Workflow Executions.
+
 Users can list Workflow Executions by any of the following:
 
 - [Workflow ID](/concepts/what-is-a-workflow-id)
 - [Workflow Type](/concepts/what-is-a-workflow-type)
 - Status
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-web-ui.md:39]:
```suggestion
- Status
- [Workflow ID](/concepts/what-is-a-workflow-id)
- [Workflow Type](/concepts/what-is-a-workflow-type)
- Start time
- End time
```
Sequence updated to match UI.

File: docs-src/concepts/what-is-the-temporal-web-ui.md [hunk 3]
```diff
@@ -31,31 +23,35 @@ The Web UI is packed with several features.
...
-- Date and Time—users can set their preferred date and time format as one of the following:
-  - UTC
-  - Local
-  - Relative
 - A [List Filter](/concepts/what-is-a-list-filter)
 
-Select a Workflow Execution to see a breakdown of the Workflow Execution's History, Workers, and pending Activities.
+Date and Time—users can set their preferred date and time format as one of the following:
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-web-ui.md:42]:
```suggestion
For start time and end time, users can set their preferred date and time format as one of the following:
```

File: docs-src/concepts/what-is-the-temporal-web-ui.md [hunk 4]
```diff
@@ -125,9 +121,9 @@ Displays the following information:
...
 ### Import Event History
 
-Navigate to `/import` to import an Event History in JSON format and populate a Workflow details page.
+Allows the Event History to be imported in JSON format for populating a Workflow details page.
 
 ### Data Encoder
 
 The Web UI can use the same Data Converter that Workers use for inputs and return values.
-The UI supports both a Remote Codec endpoint and/or the tctl plugin port.
+The UI supports both a Remote Codec endpoint and/or the `tctl` plugin port.
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-web-ui.md:129]:
```suggestion
The UI supports both a Remote Codec endpoint and the `tctl` plugin port.
```

File: docs-src/concepts/what-is-the-temporal-web-ui.md [hunk 5]
```diff
@@ -125,9 +121,9 @@ Displays the following information:
 
 ### Import Event History
 
-Navigate to `/import` to import an Event History in JSON format and populate a Workflow details page.
+Allows the Event History to be imported in JSON format for populating a Workflow details page.
```
Comment (djmagee) [inline docs-src/concepts/what-is-the-temporal-web-ui.md:124]: I'm not sure whether this feature is available via Web UI anymore.

Comment (DJSanti) [reply docs-src/concepts/what-is-the-temporal-web-ui.md:126]: Went and checked, and sure enough, it is not! I'll go change this.

**Review by djmagee — APPROVED** [review]

**Review by flossypurse — APPROVED** [review]

**Pull request merged**